pom.xml

```
@@ -86,7 +87,7 @@
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-${jenkins.baseline}.x</artifactId>
-        <version>2102.v854b_fec19c92</version>
+        <version>3867.v17b_41ce6fcb_c</version>
```

Needs to use the released version, not the incremental version of the plugin BOM.
```suggestion
        <version>3850.vb_c5319efa_e29</version>
```

---

Good catch. Not sure how we go this version

---

https://github.com/jenkins-infra/plugin-modernizer-tool/issues/534